### PR TITLE
Update urls.py to be 1.11 compatible

### PR DESCRIPTION
**Pull request by dan-gamble**

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py [hunk 1]
```diff
@@ -16,30 +19,26 @@
...
     # Basic robots.txt.
     url(r'^robots.txt$', TextTemplateView.as_view(template_name='robots.txt')),
 
     # There's no favicon here!
     url(r'^favicon.ico$', generic.RedirectView.as_view(permanent=True)),
-] + static(
-    settings.MEDIA_URL, document_root=settings.MEDIA_ROOT
-) + static(
-    settings.NODE_MODULES_URL, document_root=settings.NODE_MODULES_ROOT
-)
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: Do we not need these in 1.11?

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: As far as i can see these have never been used. I know 100% the `node_modules` one hasn't

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: I think the first is necessary for having media files in local development. I just removed it from a project and uploads stopped working.

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: ^^ By that I meant 'files that have been uploaded in the admin no longer show on the front-end', not 'uploading doesn't work'. :)

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: Ah gotcha, will add that back then 👍

**Review by danielsamuels — APPROVED** [review]

**Pull request merged**